tests/test_subset.py

```diff
@@ -1283,6 +1281,77 @@ def test_get_indexers_nd(self):
             assert ((time_var_name not in indexers.keys()) == True) #time can't be in the index
             assert (new_dataset.dims == dataset.dims)
 
+    def test_variable_type_string_oco2(self):
```

Comment:
> I just saw your comment amount the pylint error. Can you address this? You can disable specific pylint errors like so:
> 
> https://github.com/podaac/l2ss-py/blob/4a2bebdff0359d3bf1320d55ce4b3f9b386fd8d6/podaac/subsetter/subset_harmony.py#L206
> 
> We cannot leave any pylint errors or it will fail our CI/CD pipeline

pylint now 10/10